.github/workflows/publish.yml

```diff
@@ -0,0 +1,30 @@
+jobs:
+  pypi-publish:
+    name: Upload release to PyPI
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/p/pywebsocket3
+    permissions:
+      id-token: write
+    steps:
+    # Setup python envs
+    - uses: actions/checkout@v4
+    - name: Setup Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.x
+
+    # Install required dependencies
+    - name: Install dependencies
+      run: |
+        python3 -m pip install --upgrade pip
+        python3 -m pip install six yapf
+
+    # Build and create dist
+    - name: Build Package
+      run: python3 setup.py sdist
```

Comment:
We should probably migrate to `python -m build`, but there's no need to do that now. I don't if it works with Python 2.7, anyway.

Reply:
Sounds reasonable.